docs/pjrt.md

```diff
@@ -194,15 +194,65 @@ for more information.
 
 *Warning: GPU support is still highly experimental!*
 
+### Single-node GPU training
+
 To use GPUs with PJRT, simply set `PJRT_DEVICE=GPU` and configure
 `GPU_NUM_DEVICES` to the number of devices on the host. For example:
 
 ```
 PJRT_DEVICE=GPU GPU_NUM_DEVICES=4 python3 xla/test/test_train_mp_imagenet.py --fake_data --batch_size=128 --num_epochs=1
```

Comment:
Is this still accurate with `GPU_NUM_DEVICES`?

Reply:
hi @will-cromar and @jonb377 ,I know in our previous [discussion](https://github.com/pytorch/xla/pull/5657#discussion_r1357149590), we said we should replace `GPU_NUM_DEVICES` with `LOCAL_WORLD_SIZE`. But I don't think we can do that.

The reason why is if do so, then running
```
PJRT_DEVICE=GPU LOCAL_WORLD_SIZE=2 python -c 'xm.xla_device()' 
```
would hang because distributed runtime service expect 2 clients [here](https://github.com/pytorch/xla/blob/4baef3c661a66f09d6fe8af91c02884e6ef3c314/torch_xla/csrc/runtime/distributed_runtime.cc#L15-L16) but we only have 1 process/client.

How do you feel if we keep using `GPU_NUM_DEVICES` in the single-host-multi-GPU case?

Reply:
That's a good point. Let's leave `GPU_NUM_DEVICES` for single-host-multi-GPU and try to think of a better solution.

Reply:
Let's leave it in, I don't see a straightforward way around the issue. Do we need to modify the runtime initialization logic in the computation client to account for this?